From 724e708744606d4b8e2a8e5732730e2ff368ff0c Mon Sep 17 00:00:00 2001
From: Avula Sri Charan <quic_asrichar@quicinc.com>
Date: Thu, 27 Oct 2022 10:52:12 +0530
Subject: [PATCH] Changes to disable compilation errors in 32 bit

In yocto builds signcontext-32.h will be included for 32
bit builds and signcontext-64.h for 64 bit builds because
of the absence of linux/types.h in signcontext-32.h we will
face the error __s8 undfined.So adding typedef of __s8 in
linux_wext.h.

Signed-off-by: Avula Sri Charan <quic_asrichar@quicinc.com>
---
 src/drivers/linux_wext.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/drivers/linux_wext.h b/src/drivers/linux_wext.h
index e7c7001e1..9cfebb02d 100644
--- a/src/drivers/linux_wext.h
+++ b/src/drivers/linux_wext.h
@@ -26,6 +26,7 @@ typedef int32_t __s32;
 typedef uint16_t __u16;
 typedef int16_t __s16;
 typedef uint8_t __u8;
+typedef __signed__ char __s8;
 #ifndef __user
 #define __user
 #endif /* __user */
-- 
2.17.1

